classroom.rb

```diff
@@ -8,7 +8,7 @@ def initialize(label)
     @students = []
   end
 
-  def many_students(student)
+  def add_student(student)
     @students << student
     student.Classroom = self
```

Comment:
[OPTIONAL] I guess you meant to write `student.classroom` not `student.Classroom`

Reply:
oh! thanks!